contract_sale_generation/tests/test_contract_sale.py

```diff
@@ -96,6 +96,11 @@ def test_cron_recurring_create_sale(self):
         self.contract_line.recurring_invoicing_type = "post-paid"
         self.contract_line.date_end = "2020-03-15"
         self.contract_line._onchange_is_auto_renew()
+        # If we do not recompute recurring_next_date
+        # then it maintains it's 'old' value.
+        # TODO: Research that
+        recurring_next_date = self.contract_line.recurring_next_date
+        self.assertGreaterEqual(recurring_next_date, self.contract_line.date_start)
```

Comment:
Added these 2 lines in order to make unit tests work, and pre-commit to not complain. Need to see why this is happening though.